test/suites/standard1/configuration.robot

```diff
@@ -118,37 +118,33 @@ Deploy MicroShift Without CSI Snapshotter
     ...    Remove Storage Drop In Config
     ...    Restart MicroShift
 
-Custom TLS 1_3 configuration
-    [Documentation]    Configure API server to use TLS 1.3 and verify only that
-    ...    version works
-    [Setup]    Setup TLS Configuration    ${TLS_13_MIN_VERSION}
+Custom TLS 1_2 configuration
+    [Documentation]    Configure a custom cipher suite using TLS 1.2 and verify it is used
+    [Setup]    Setup TLS Configuration    ${TLS_12_CUSTOM_CIPHER}
 
-    ${rc}=    Execute Command
-    ...    openssl s_client -connect ${USHIFT_HOST}:6443 -tls1_3 <<< "Q"
-    ...    sudo=True    return_stdout=False    return_stderr=False    return_rc=True
+    ${stdout}    ${rc}=    Openssl Connect Command    -tls1_2
     Should Be Equal As Integers    ${rc}    0
+    Should Contain    ${stdout}    TLSv1.2, Cipher is ECDHE-RSA-AES128-GCM-SHA256
```

Comment:
we are configuring a different suite in line 59, but what comes back is one of the defaults (TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256).
Instead of letting the cipher suite to be chosen automatically, should we specify it in the openssl command? even if the returned value is correct we are not testing whether the configure suite is used.

Reply:
I agree, I pushed https://github.com/openshift/microshift/pull/4479/commits/24bbc7691dde91ff2b6243b5eddac2c1d00f6831 to check every cipher individually on TLS v1.2 with `openssl` commands. Also, modified the custom cipher suite for TLS v1.2.